rites/roman1969/build/build.ts

```diff
@@ -1,6 +1,9 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable import/no-unused-modules */
```

Comment:
these are things we can configure in config/eslintrc, i.e. allowing `devDependencies` in `/build` folders...

ex: 
```js
    'import/no-extraneous-dependencies': [
      'error',
      { devDependencies: ['build/**/*.ts'] },
    ],
```